src/docs/product/sentry-basics/search/searchable-properties.mdx

```diff
@@ -0,0 +1,74 @@
+
+### Issue Properties
+
+[Issues](/product/issues/) are an aggregate of one or more events. Searchable properties include workflow status, assignment, aggregate counts, and age.
+
+Below is a list of issue-level keys and tokens reserved and known to Sentry:
+
+| Key/Token                    | Description |
+|--------------------------|-------------|
+| age | Restrict results to issues created since `age`. The syntax is similar to the Unix `find` command. Supported suffixes: `m -> minutes`, `h -> hours`, `d -> days` `w -> weeks` |
+| age:-24h | Return issues that are new in the last 24 hours. |
+| age:+12h | Return issues older than 12 hours. |
+| age:+12h age:-24h | Return issues created between 12 and 24 hours ago. |
+| assigned | Filter on the user which the issue is assigned to. Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee, or `#team-name`. |
+| assigned_or_suggested | Filter on the user or team to which the issue is assigned or suggested to be assigned. Suggested assignees are determined by matching [ownership rules](/product/error-monitoring/issue-owners/) and [suspect commits](/product/releases/suspect-commits/). Values can be a user ID (your email address), `me` for yourself, `me_or_none` for yourself or no assignee/suggestions, or `#team-name`. |
+| bookmarks | Filter on the user which the issue is bookmarked by. Values can be your user ID (your email address) or `me` for yourself. |
+| first-release | Restrict results to issues first seen within the given release. Exact match on the version of a release, or `first-release:latest` to pick the most recent release. |
+| has | Restrict results to issues which have _any_ value for a tag. |
+| has:user | Restrict results to issues which have a user value for a tag. |
+| is | Filter on various properties of an issue. |
+| is:unresolved<br /> is:resolved<br /> is:ignored | Filter on the state of an issue. |
+| is:assigned<br /> is:unassigned | Return issues based on whether they are are assigned or not. |
+| is:linked<br /> is:unlinked | Return issues based on whether they are linked (to an external issue tracker) or not. |
+| lastSeen | Restrict results to issues that were last seen since or until a given point in time. Usage is similar to the `age` token (see above). |
+| lastSeen:+30d | Return issues last seen 30 days ago or more. |
+| lastSeen:-2d | Return issues last seen within the last two days. |
+| timesSeen | Restrict results to issues that have been seen exactly, at least, or at most some number of times.<br /><br /> An exact match token: `timesSeen:10`.<br /><br /> Upper or lower bounds tokens: `timesSeen:>10`, `timesSeen:>=10`, `timesSeen:<10`, `timesSeen:<=10` |
+
+
+
+### Event Properties
+
+Events are the underlying event data captured using Sentry SDKs (read: errors and exceptions).
```

Comment:
Can you please verify that the events definition here does not include transactions? I think it should.